scripts/tcl_commands/lvs.tcl

```diff
@@ -93,20 +99,16 @@ proc run_lvs {{layout "$::env(EXT_NETLIST)"}} {
     # GDS LVS uses STD_CELL_LIBRARY spice and
     # if defined, additional LVS_EXTRA_STD_CELL_LIBRARY spice and LVS_EXTRA_GATE_LEVEL_VERILOG files
     # Write Netlist
+    puts "Starting LVS"
     if { $::env(LVS_INSERT_POWER_PINS) } {
-        set powered_netlist_name [index_file $::env(signoff_tmpfiles)/powered_netlist.v]
-        set powered_def_name [index_file $::env(signoff_tmpfiles)/powered_def.def]
+        set powered_netlist_name [index_file $::env(signoff_tmpfiles)/$::env(DESIGN_NAME).pnl.v]
+        set powered_def_name [index_file $::env(signoff_tmpfiles)/$::env(DESIGN_NAME).p.def]
+
         write_powered_verilog\
             -output_verilog $powered_netlist_name\
             -output_def $powered_def_name\
-            -log $::env(signoff_logs)/write_verilog.log\
-            -def_log $::env(signoff_logs)/write_powered_def.log
-
-        set_netlist $powered_netlist_name
-
-        if { $::env(LEC_ENABLE) } {
-            logic_equiv_check -rhs $::env(PREV_NETLIST) -lhs $::env(CURRENT_NETLIST)
-        }
+            -log $::env(signoff_logs)/write_powered_nl.log\
+            -def_log $::env(signoff_logs)/write_powered_df.log
```

Comment:
`write_powered_df.log`

Reply:
yes?

Reply:
sorry, small typo `write_powered_def.log`

Reply:
does it matter

Reply:
ill fix it anyway